src/models/__tests__/profile.test.ts

```diff
@@ -45,40 +45,71 @@ const aRetrievedProfile: RetrievedProfile = {
   ...aStoredProfile
 };
 
-describe("findLastVersionByModelId", () => {
-  it("should resolve to an existing profile", async () => {
-    const containerMock = ({
-      items: {
-        create: jest.fn(),
-        query: jest.fn(() => ({
-          fetchAll: jest.fn(() =>
-            Promise.resolve({
-              resources: [aRetrievedProfile]
-            })
-          )
-        }))
-      }
-    } as unknown) as Container;
-
-    const model = new ProfileModel(containerMock);
+const aRetrievedProfileWithEnabledReminder: RetrievedProfile = {
```

Comment:
Is this now unused?

Reply:
right, removed in https://github.com/pagopa/io-functions-commons/pull/290/commits/da0599d241e476b31ca5f5fc79b8f222d78da499